src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework.mdx

```diff
@@ -931,12 +931,34 @@ Prior versions of Npgsql may also be instrumented, but duplicate and/or missing
           <td>
 If your application uses ADO.NET, you can install a combination of New Relic agents that will generate an ADO dashboard. Here’s what you need to do:
 
-1. Install the New Relic APM .NET agent and the infrastructure agent by following the steps in our [installation tool](https://one.newrelic.com/marketplace?state=bf68ca4b-b7cc-3227-fcc3-3349f5aa92f6).
+1. Install the New Relic APM .NET agent and the infrastructure agent by following the steps in our [guided installation](https://one.newrelic.com/marketplace?state=bf68ca4b-b7cc-3227-fcc3-3349f5aa92f6).
 2. Install the Microsoft Windows Perfmon/WMI on-host integration for New Relic infrastructure by [following the nri-perfmon instructions](https://github.com/newrelic/nri-perfmon#nri-perfmon---windows-perfmonwmi-on-host-integration-for-new-relic-infrastructure).
 3. If you installed Perfmon manually, [restart your infrastructure agent](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent).
 4. Go to our ADO.NET quickstart page to [install the ADO.NET dashboards](https://newrelic.com/instant-observability/ado-dotnet).
           </td>
         </tr>
+        <tr>
+          <td>
+            Blazor WebAssembly
+          </td>
+          <td>
+            To monitor Blazor WebAssembly, you need to install the .NET agent, then install the .NET quickstart from our instant observability page. Here's what you need to do:
```

Comment:
To monitor Blazor Server, you need to install the .NET agent, then install the .NET quickstart from our instant observability page. Here's what you need to do: